pizza-facts-&-trivia.md

```diff
@@ -23,6 +23,7 @@ We've categorized the facts into different sections to help explore the diverse
 - An International Pizza Expo takes place each year in Las Vegas, Nevada. It’s the world’s largest gathering of members from the pizza industry. Unfortunately, it’s not open to the public. You have to be in the pizza business to attend.
 - As of 2017, Domino’s Pizza is the most popular pizza chain in the world. The company has more than 17,000 pizza stores in 90 countries, and this number continues to grow.
 - Anchovies are America's least favorite topping.
+- The longest pizza ever made was over 6,300 feet long. This record-breaking pizza was created in Naples, Italy, in 2016. It required over two tons of flour, 1.5 tons of tomato sauce, and 1.5 tons of mozzarella cheese to make.
```

Comment:
Could please check these numbers again I thought these numbers weren't correct.

Reply:
Yes i just confirmed now and you are correct. Thank you @Sunny-unik .
I will push the updated numbers asap